tests/e2e/common/common.go

```diff
@@ -73,18 +73,23 @@ type TestCase struct {
 
 // GetVersionsFromEnv will return values from required environment variables,
 // if environment variables are not set it fails the test.
-func GetVersionsFromEnv(t *testing.T) (string, string) {
+func GetVersionsFromEnv(t *testing.T, latest bool) (string, string) {
```

Comment:
Maybe we can create another function `GetLatestVersionsFromEnv` instead. If not, let's update the docs comment to mention what it does.

Reply:
Will update the doc